indico/modules/logs/templates/entry_email.html

```diff
@@ -75,6 +75,20 @@
             <p style="white-space: pre-wrap; word-break: break-word;">{{ data.subject }}</p>
         </td>
     </tr>
+    <tr class="i-table content">
+        <td class="i-table caption log-caption">
+            {% trans %}Attachments{% endtrans %}
+        </td>
+        <td class="i-table value searchable">
+            {% if data.attachments %}
+                {{ data.attachments|join(', ') }}
+            {% else %}
+                <span class="text-muted">
+                    {% trans %}No attachments{% endtrans %}
+                </span>
+            {% endif %}
+        </td>
+    </tr>
```

Comment:
Should this row be shown at all when there are no attachments?

Reply:
Id say it's nice for the UI to be consistent in this case.. if you're scrolling through many email logs it might get annoying to see a row popping on and off